static/js/settings_bots.js

```diff
@@ -317,18 +317,41 @@ exports.set_up = function () {
                 });
             },
         });
+    });
 
-
+    $("#active_bots_list").on("click", "button#download_bot_zuliprc", function () {
+        var bot_info = $(this).closest(".bot-information-box");
+        var email = bot_info.find(".email .value").text();
+        var api_key = bot_info.find(".api_key .api-key-value-and-button .value").text();
+        var zuliprc_uri = exports.generate_zuliprc_uri($.trim(email), $.trim(api_key));
+        var anchor = document.createElement('a');
+        document.body.appendChild(anchor);
+        anchor.download = "zuliprc";
+        anchor.href = zuliprc_uri;
+        anchor.click();
     });
 
-    $("#active_bots_list").on("click", "a.download_bot_zuliprc", function () {
+    $("#active_bots_list").on("click", "button.open_get_zuliprc_modal", function () {
         var bot_info = $(this).closest(".bot-information-box");
         var email = bot_info.find(".email .value").text();
         var api_key = bot_info.find(".api_key .api-key-value-and-button .value").text();
+        var zuliprc_text = exports.generate_zuliprc_content($.trim(email), $.trim(api_key));
```

Comment:
You probably should use `email.trim()` and `api_key.trim()` here.  The need for `$.trim` is pretty ancient.